rust/src/http2/http2.rs

```diff
@@ -350,6 +402,7 @@ pub struct HTTP2State {
     transactions: Vec<HTTP2Transaction>,
     progress: HTTP2ConnectionState,
     pub files: Files,
+    flow: *const Flow,
```

Comment:
ideally we're not adding this if we can just pass it around as a function argument

Reply:
Right